content/docs/static-type-checking.md

```diff
@@ -24,7 +24,7 @@ yarn add --dev babel-preset-flow
 npm install --save-dev babel-preset-flow
 ```
 
-Then add `flow` to your Babel presets config.
+Then add `flow` to your Babel presets config in`.babelrc`.
```

Comment:
There should be a space between "in`.babelrc`."

I'm going to just modify this myself directly.